### PR TITLE
fix(ui): web console to load regardless of pgwire disabled

### PR DESCRIPTION
**Pull request by glasstiger**
Closes https://github.com/questdb/questdb/issues/4483

web console to load regardless of pgwire disabled

### CODE REVIEW

**Review by ideoma — DISMISSED** [review]

File: core/src/main/java/io/questdb/PropServerConfiguration.java [hunk 1]
```diff
@@ -886,11 +886,11 @@ public PropServerConfiguration(
                 this.pgNamedStatementCacheCapacity = getInt(properties, env, PropertyKey.PG_NAMED_STATEMENT_CACHE_CAPACITY, 32);
                 this.pgNamesStatementPoolCapacity = getInt(properties, env, PropertyKey.PG_NAMED_STATEMENT_POOL_CAPACITY, 32);
                 this.pgPendingWritersCacheCapacity = getInt(properties, env, PropertyKey.PG_PENDING_WRITERS_CACHE_CAPACITY, 16);
-
-                this.forceSendFragmentationChunkSize = getInt(properties, env, PropertyKey.DEBUG_FORCE_SEND_FRAGMENTATION_CHUNK_SIZE, Integer.MAX_VALUE);
-                this.forceRecvFragmentationChunkSize = getInt(properties, env, PropertyKey.DEBUG_FORCE_RECV_FRAGMENTATION_CHUNK_SIZE, Integer.MAX_VALUE);
             }
 
+            this.forceSendFragmentationChunkSize = getInt(properties, env, PropertyKey.DEBUG_FORCE_SEND_FRAGMENTATION_CHUNK_SIZE, Integer.MAX_VALUE);
+            this.forceRecvFragmentationChunkSize = getInt(properties, env, PropertyKey.DEBUG_FORCE_RECV_FRAGMENTATION_CHUNK_SIZE, Integer.MAX_VALUE);
```
[the remark this answers is not in the document]
Comment (glasstiger) [reply core/src/main/java/io/questdb/PropServerConfiguration.java]: done

[the remark this answers is not in the document]
Comment (glasstiger) [reply core/src/main/java/io/questdb/PropServerConfiguration.java]: although it is debug config, still nicer to have it separated

**Review by ideoma — DISMISSED** [review]

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 14 / 14 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/PropServerConfiguration.java|10|10|100.00%|
|:large_blue_circle:|io/questdb/PropertyKey.java|4|4|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 14 / 14 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/PropServerConfiguration.java|10|10|100.00%|
|:large_blue_circle:|io/questdb/PropertyKey.java|4|4|100.00%|

**Review by bluestreak01 — APPROVED** [review]

**Pull request merged**